operator/v1/types_network.go

```diff
@@ -328,6 +331,16 @@ type HybridOverlayConfig struct {
 	HybridOverlayVXLANPort *uint32 `json:"hybridOverlayVXLANPort,omitempty"`
 }
 
+type HardwareOffloadConfig struct {
+	// enableHardwareOffload specifies whether or not OVS hardware offload
+	// should be enabled. If unset, this property defaults to 'false' and
+	// OVS hardware offload is disabled.
+	EnableHardwareOffload *bool `json:"enableHardwareOffload"`
+	// hardwareOffloadPolicy is the Traffic Control Policy (tc-policy) specified
+	// for OVS instance. If unset, this property defaults to 'none'.
+	HardwareOffloadPolicy string `json:"hardwareOffloadPolicy"`
```

Comment:
In the docs for this, can you give some functional examples? Or is this too complicated for a docblock?

Reply:
> Does it make sense to add this to Openshift-SDN as well?

OpenShift-SDN doesn't support ovs hw offload yet, I don't know any plan of doing this.

Reply:
> In the docs for this, can you give some functional examples? Or is this too complicated for a docblock?

I will add some examples.

Reply:
added

Reply:
In general, we would like to avoid free-form strings if an invalid value can take down a cluster. So, we will probably need to validate this. Would you like to define the set of supported values in the type or in CNO?

The advantage of defining this in the type is that invalid values cannot even be committed to etcd. The disadvantage is that, if there is another mode we forgot to add, the customer is unable to use it.

Reply:
> In general, we would like to avoid free-form strings if an invalid value can take down a cluster. So, we will probably need to validate this. Would you like to define the set of supported values in the type or in CNO?

sure, added the `TcPolicy` type.